stable/cachet/templates/deployment.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: {{ include "cachet.fullname" . }}
   labels:
 {{ include "cachet.labels" . | indent 4 }}
+{{- if .Values.extraLabels }}
```

Comment:
You're only adding them to the deployment and pods? I think you should put it in the `_helpers.tpl` so that `extraLabels` is included in the `labels` template function, otherwise it's not very consistent?